metrics/foundations_proposed_migration_by_team.py

```diff
@@ -20,7 +20,7 @@ def get_proposed_migration_queue(team):
             logging.error('URL %s failed with code %u', req.geturl(), code)
             return {}
         yamldata = StringIO(req.read().decode('UTF-8'))
-    yaml_handle = yaml.safe_load(yamldata)
+    yaml_handle = yaml.load(yamldata)
```

Comment:
We could have this continue working on later versions by specifying `Loader=yaml.Loader` here, I think, as ["UnsafeLoader [is] also called Loader for backwards compatability"](https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation#how-to-disable-the-warning).